docs/installation.md

```diff
@@ -12,6 +12,61 @@
 
 Follow detailed instructions in separate [Domain Protect Deploy](https://github.com/domain-protect/domain-protect-deploy) repository with GitHub Actions deployment workflow
 
+## Atlantis
+
+* For atlantis to work, you can simply create a new stack (folder) in your environment with the name `domain-protect`. You will then make the necessary changes to the `atlantis.yaml` file, so atlantis knows where all the terraform code is from.
+
+Example changes to `atlantis.yaml`:
+
+```yaml
+- name: domain-protect-security-audit
+  dir: security/domain-protect
+  workspace: security-audit
+  workflow: security-audit
+  terraform_version: 1.3.1
+```
+
+* In the `domain-protect` folder, you can copy the content of `main.tf` file from this repo. For each of the module, make sure you change the source to this repo so you can reuse the module here
+
+For example, update:
+
+```terraform
+module "kms" {
+  source  = "./terraform-modules/kms"
+  project = var.project
+  region  = var.region
+}
+```
+
+to
+
+```terraform
+module "kms" {
+  source  =  "git::https://github.com/domain-protect/domain-protect.git//terraform-modules/kms"
+  project = var.project
+  region  = var.region
+}
+```
+
+Also, because of the current way the lambda code modules are set up, you will need to copy the contents of `scripts/lambda-build`, `build` and `lambda_code` folders with the same path to your `domain-protect` folder. In the end, you will have a structure like this:
```

Comment:
suggest removing the word "current" - if we do restructure the code base in the future, this section can be updated at that time

Reply:
updated this